submissions/YarHadi/html-css-popup/index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>POPUP</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <ul class="nav-menu">
```

Comment:
this is a navigation menu 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav?retiredLocale=uk

Reply:
But no need to remove a header, just add a nav))

Reply:
In the top of the site we have a header, which contains nav menu on it. Think about it this way

Reply:
Sorry. My bad. 
Fixed